src/Maestro/Client/src/Microsoft.DotNet.Maestro.Client.csproj

```diff
@@ -1,15 +1,23 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard1.4;$(NetFxTfm)</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;$(NetFxTfm)</TargetFrameworks>
     <IsPackable>true</IsPackable>
     <Description>This package provides access to the Maestro Api</Description>
     <LangVersion>7.1</LangVersion>
+
+    <SwaggerOutputDirectory>$(MSBuildProjectDirectory)\Generated</SwaggerOutputDirectory>
+    <SwaggerClientName>MaestroApi</SwaggerClientName>
+    <SwaggerDocumentUri>https://maestro-int.westus2.cloudapp.azure.com/api/swagger.json</SwaggerDocumentUri>
```

Comment:
Little weird that this will essentially always be 1 version behind... but maybe that's ok.

Reply:
Yea, technically we can `dotnet run` the website then get it live from `http://localhost:8080/api/swagger.json` but I don't think we need to do that for this change.